### PR TITLE
Feature: Landing page

### PR DESCRIPTION
**Pull request by jtingling**
### What this PR does (required):
- Responsive landing page. Logged out or new visitors will be redirected to `/welcome`. Includes `Formik` scaffolding.

#### Desktop:
DesktopView
#### Tablet:
TabletView
#### Mobile:
MobileView

### Any information needed to test this feature (required):
- Enter command `npm run start` in the `client` directory.

### Any issues with the current functionality (optional):
- Form elements are wrapped with `Formik` for future implementation. `handleSubmit` returns `null` as there is no backend controllers to handle the form data yet.
- I'll update this PR if it needs to be removed.

### CODE REVIEW

**Review by ezeilo-su — COMMENTED** [review]
Good progress so far!

File: client/src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -88,7 +88,13 @@ const MenuItem: React.FC<{
 
   return (
     <Grid key={resource} sx={{ textAlign: 'center' }} xs={2} justifySelf="flex-end" item>
-      <NavLink className={classes.navbarItem} to={resource}>
+      <NavLink
+        className={clsx(
+          classes.navbarItem,
+          location.pathname === '/welcome' && item === 'Become a sitter' && classes.navSitterButton,
```
Comment (moffatethan) [inline client/src/components/Navbar/Navbar.tsx:94]: Let's make this the index route please

File: client/src/components/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -130,7 +136,7 @@ const Navbar: React.FC = () => {
 
   return (
     <Grid
-      className={clsx(classes.navbar, location.pathname === '/' && classes.transparentNavbar)}
+      className={clsx(classes.navbar, location.pathname === '/welcome' && classes.transparentNavbar)}
```
Comment (moffatethan) [inline client/src/components/Navbar/Navbar.tsx:139]: Should be the index route

File: client/src/context/useAuthContext.tsx [hunk 3]
```diff
@@ -68,7 +68,7 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
           // don't need to provide error feedback as this just means user doesn't have saved cookies or the cookies have not been authenticated on the backend
           setLoggedInUser(null);
           if (!(history.location.pathname === '/signup')) {
-            history.push('/login');
+            history.push('/welcome');
```
Comment (moffatethan) [inline client/src/context/useAuthContext.tsx:71]: Should be index route

File: client/src/pages/Landing/FindSitter/FindSitter.tsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import { useStyles } from './useStyles';
+import { Grid, Box, Typography, Button, CircularProgress } from '@mui/material';
+import FormInput from '../../../components/FormInput/FormInput';
+
+interface Props {
```
Comment (moffatethan) [inline client/src/pages/Landing/FindSitter/FindSitter.tsx:7]: `FindSitterProps` let's use better naming here

File: client/src/pages/routes/route.ts [hunk 5]
```diff
@@ -5,8 +5,16 @@ import SignUp from '../SignUp/SignUp';
 import { AccountType } from '../../types/AccountType';
 import NotFound from '../NotFound/NotFound';
 import Settings from '../Settings/Settings';
+import Landing from '../Landing/Landing';
 
 const menuItems = [
+  {
+    item: 'Landing',
+    resource: '/welcome',
```
Comment (moffatethan) [inline client/src/pages/routes/route.ts:13]: Let's rename to index route

**Review by moffatethan — APPROVED** [review]

**Pull request merged**